README.md

```diff
@@ -11,19 +11,23 @@ npm install @brinkninja/sdk
 ## Setup
 
 ```
+const ethers = require('ethers)
 const brinkSDK = require('@brinkninja/sdk')
 
-// Instantiate with environment config: `local`, `dev`, `prod` (see [brink-environment](https://github.com/brinktrade/brink-environment) and an instance of [ethers.js](https://github.com/ethers-io/ethers.js/)
-const brink = brinkSDK({
-  environment: 'dev',
-  ethers
-})
+// Obtain a signer and a provider from the Ethers Library [link above]
+const ethersProvider = await ethers.getDefaultProvider(process.env.RPC_URLS)
+const ethersSigner = await new ethers.Wallet(process.env.ETH_PRIVATE_KEY)
```

Comment:
this section should probably be removed, since there's no explanation of what these env vars are. maybe just include some docs links from ethers.js below to clarify what `ethersProvider` and `ethersSigner` need to be